mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
```

Comment:
The status code 404 is typically used for "Not Found" errors, whereas 400 is used for "Bad Request" errors. In this case, since the error is "Order not found", 404 is the correct status code. The change to 400 is incorrect and should be reverted.